src/theory/quantifiers/ceg_t_instantiator.cpp

```diff
@@ -917,7 +917,7 @@ Node BvInstantiator::hasProcessAssertion(CegInstantiator* ci, SolvedForm& sf,
   Node atom = lit.getKind() == NOT ? lit[0] : lit;
   bool pol = lit.getKind() != NOT;
   Kind k = atom.getKind();
-  if (pol && k == EQUAL) {
+  if ((pol && k == EQUAL) || (k != EQUAL && options::cbqiBvInvLt())) {
```

Comment:
This is correct, however, I'll probably want to make a 3 valued option since setting cbqiBvInvLt to true makes cbqiBvSlackIneq unused.  It'd be better to have an option like cbqiBvIneqPolicy= { EQ_SLACK, EQ_BOUNDARY, UNCHANGED } or something like that.

Reply:
Btw, I can make this change (adding "mode" options takes a bit of getting used to)

Reply:
I agree, that's a good idea. Let's change that after this PR is merged in.